### DISCUSSION TITLE
How are we using `etl_group`?

### DISCUSSION THREAD

**Discussion by bendnorman** [catalyst-cooperative/pudl#3225] [Ideas]
It seems like the meaning of the `etl_group` attribute of resources objects might be outdated? I think it would be nice to align our notion of `etl_group` with our Asset Groups. Asset groups right now are determined by the resource/assets's datasource and processing layer: `{layer}_{datasource}`.

Why did we originally create `etl_group`? It's being used in various places as a way to filter out groups of assets we might not want metadata for.

**Comment by zaneselvans** [comment]
It was largely a convenience -- just a way to address various groups of tables that needed to be iterated over in the ETL rather than compiling lists of them.  If there's a better way to group assets now, then we can use that instead, but I doubt that our current asset groups map cleanly to the structural similarities that were reflected by `etl_group` and IIRC actually accessing the list of assets in an asset group was kind of a pain in the ass.
